### PR TITLE
Added entries in navigation.yml, added openprinting logo and added contributing page.

### LINKED ISSUE #8: Create the Menu Bar:
List: 

Home Page
Downloads
Tutorials
Project List
Blogs
Contact
Donation
News / Events
Driverless Printing

### PR DESCRIPTION
**Pull request by dheeraj135**
Closes #8 #13 .
Preview for issue 8 and 13:
screenshot from 2019-01-21 15-57-56

### CODE REVIEW

**Comment by dheeraj135** [comment]
Added some work on #17  . Contributing web page and Contributing.md still needs to be worked on. 
Link: https://dheeraj135.github.io/contribute-website/

**Comment by sahilarora535** [comment]
@dheeraj135 Great work, the #13 issue has been done nicely, however, I have added some comments on #8, please have a look.

Also, which adding merge request from next time, please follow these best practices:
1. One pull request should ideally aim to fix one issue if the issues are independent. If the issues are dependent, then it is okay to have one merge request only. Like here, this merge request for the title bar is okay.
2. Squash commits before making a merge request. For instance, you have 3 commits with the message "Added Contributing.md". This should ideally be a separate pull request, and only one commit.
3. See how to write good commit messages: https://chris.beams.io/posts/git-commit/ (very very important).

Once all this is good, I will merge the branch.

**Comment by dheeraj135** [comment]
@sahilarora535 I have changed menu bar according to your comment. Here: [github](https://dheeraj135.github.io/).
Thanks for your input. I will keep these points in mind.

File: _data/navigation.yml [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
-    url: /collection-archive/
-  - title: "Contribute"
-    url: /sitemap/
-
+    url: /downloads/
+  - title: "Driverless Printing"
+    url: /driverless/
+  - title: "Contact Us"
+    url: /contact/
+  - title: "Donation"
```
Comment (sahilarora535) [inline _data/navigation.yml]: Please change "Donation" to "Donations"
also, it would make sense to change the corresponding link to "donations"

**Pull request merged**